USER_MANUAL.md

```diff
@@ -1,12 +1,242 @@
 K User Manual
 =============
 
-**Under Construction**
+**NOTE:** The K User Manual is still **under construction**; some features of K
+may have partial or missing documentation.
+
+Introduction
+------------
+
+### Why K?
+
+The K Framework is a programming language and system design toolkit made for
+practioners and researchers alike.
+
+**K For Practioners:**
+*K is a framework for deriving programming languages tools from their semantic
+specifications.*
+
+Typically, programming language tool development follows a similar pattern.
+After a new programming language is designed, separate teams will develop
+separate language tools (e.g. a compiler, interpreter, parser, symbolic
+execution engine, etc). Code reuse is uncommon. The end result is that for each
+new language, the same basic tools and patterns are re-implemented again and
+again.
+
+K approaches the problem differently, by imagining how different tools could be
+derived generically from a common specification. In other words, for any
+language, a language-generic tool is a function that takes a language spec as
+input and produces a tool as output. The end result is that the exercise of
+designing new languages and their associated tooling is now reduced to
+developing a single language specfication from which we derive our tooling *for
+free*.
```

Comment:
It's a little confusing that the tools are both "derived generically from a common specification" and that they take the "language spec as input".

```suggestion
K approaches the problem differently -- it generates each of these tools from a single language specification.
The work of programming language design and tool implementation are made separate concerns.
The end result is that the exercise of
designing new languages and their associated tooling is now reduced to
developing a single language specification from which we derive our tooling *for
free*.
```